cpp/data/installer/ub_tools.sql

```diff
@@ -41,20 +41,20 @@ CREATE TABLE delivered_marc_records (
     hash VARCHAR(40) NOT NULL,
     zeder_id VARCHAR(10) NOT NULL,
     delivered_at TIMESTAMP NOT NULL DEFAULT NOW(),
-    journal_name VARCHAR(191) NOT NULL,
-    main_title VARCHAR(191) NOT NULL,
+    journal_name VARCHAR(1000) NOT NULL,
+    main_title VARCHAR(1000) NOT NULL,
```

Comment:
what about the field length in other similar tables? e.g.
- delivered_marc_records_superior_info.title
- metadata_presence_tracer.journal_name and metadata_field_name
- rss_aggregator.item_id